src/hooks/useNotificationTypes.tsx

```diff
@@ -660,11 +659,13 @@ export const notificationTypes: NotificationTypeConfig[] = [
     useTrigger: ({ trigger }) => {
       const { dydxAddress } = useAccounts();
       const { getInTouch } = useURLConfigs();
-      const feedbackRequestWalletAddresses = useEnvConfig('feedbackRequestWalletAddresses');
+      const feedbackRequestWalletAddresses = useStatsigDynamicConfigValue(
```

Comment:
so if some rando were to host the frontend and not configure statsig what would happen?

Reply:
ah good point - it errors out though doesn't crash - lemme add a fix